core/src/main/java/tech/tablesaw/table/TableSlice.java

```diff
@@ -147,8 +187,20 @@ public Table asTable() {
     return table;
   }
 
-  private IntIterator intIterator() {
-    return selection.iterator();
+  /**
+   * IntIterator of source table row numbers that are present in this view. This can be used to
+   * in combination with the source table to iterate over the cells of a column
+   * in a sorted order without copying the column.
+   *
+   * @return an int iterator of row numbers in the source table that are present in this view.
+   */
+  public PrimitiveIterator.OfInt sourceRowNumberIterator() {
```

Comment:
Should this be `public`? E.g. it seems like it could be `protected`

Reply:
Now that you mention it I think it can be private. The method it replace was private. I will send a PR.